### PR TITLE
Refactor transaction container

### PR DESCRIPTION
**Pull request by taojintianxia**

### CODE REVIEW

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/DockerStorageContainer.java [hunk 1]
```diff
@@ -110,24 +111,30 @@ protected void postStart() {
...
         result.setPassword(getUnifiedPassword());
         result.setMaximumPoolSize(4);
         result.setTransactionIsolation("TRANSACTION_READ_COMMITTED");
         return result;
     }
     
+    /**
+     * Get JDBC URL.
+     * 
+     * @param dataSourceName datasource name.
```
Comment (terrymanu) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/DockerStorageContainer.java:125]: Please remove `.`

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/framework/container/config/mysql/MySQLContainerConfigurationFactory.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import java.util.Map;
+
+/**
+ * MySQL container configuration factory.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class MySQLContainerConfigurationFactory {
+    
+    /**
+     * Create new instance of mysql container configuration.
```
Comment (terrymanu) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/framework/container/config/mysql/MySQLContainerConfigurationFactory.java:35]: mysql should be MySQL

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/framework/container/config/opengauss/OpenGaussContainerConfigurationFactory.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import java.util.Map;
+
+/**
+ * OpenGauss container configuration factory.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class OpenGaussContainerConfigurationFactory {
+    
+    /**
+     * Create new instance of opengauss container configuration.
```
Comment (terrymanu) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/framework/container/config/opengauss/OpenGaussContainerConfigurationFactory.java:34]: please rename OpenGauss to openGauss

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/DockerStorageContainer.java (hunk 1 above)
Comment (taojintianxia) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/DockerStorageContainer.java:125]: done

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/framework/container/config/opengauss/OpenGaussContainerConfigurationFactory.java (hunk 3 above)
Comment (taojintianxia) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/framework/container/config/opengauss/OpenGaussContainerConfigurationFactory.java:34]: I will refactor the `opengauss` in doc .

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/framework/container/config/mysql/MySQLContainerConfigurationFactory.java (hunk 2 above)
Comment (taojintianxia) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-transaction/src/test/java/org/apache/shardingsphere/integration/transaction/framework/container/config/mysql/MySQLContainerConfigurationFactory.java:35]: done

**Review by terrymanu — APPROVED** [review]

**Pull request merged**